lib/tags/group-settings.tag

```diff
@@ -204,6 +207,7 @@
                                 self.group.description = description;
                                 self.refs.description.classList.remove('graphjs-error');
                                 self.refs.description.classList.add('graphjs-success');
+                                self.refs.description.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
```

Comment:
Please remove this change. self.refs.description refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above